### PR TITLE
adds cooldown for csaber noise

### PR DESCRIPTION
**Pull request by zjdtmkhzt**
## About the PR 
Adds a cooldown for the csaber noise, because some people said that spamming that is annoying and they are kinda right.

Not sure if PROC_ON_COOLDOWN is ok to use here, but I think so? It works well in my testing!
I wanted to use it, since I've never gotten to until now. :D
## Why's this needed? 
Less annoying noise is good!

### CODE REVIEW

**Comment by ZeWaka** [comment]
@pali6 is this correct usage?

**Comment by pali6** [comment]
No, PROC_ON_COOLDOWN is more for when there's only a single obvious cooldown the proc can be on (i.e. it's a verb and the cooldown is using that verb). Use a named cooldown with ON_COOLDOWN instead here.

**Comment by zjdtmkhzt** [comment]
Right, thanks, pali, I see what you mean!

**Comment by pali6** [comment]
Looks good to me assuming you want separate cooldowns for on and off. (Though a minor nitpick - `SECOND` should probably be `SECONDS` if you want it to be 2.)

**Comment by zjdtmkhzt** [comment]
Yeah, I do want two separate ones, it's actually nicer than the previous thing I had, because it now does the swoosh and swish noise both, even if they do it quickly, they just can't spam it.
And yeah, I guess I can't plural today, but it still works. :p

File: code/obj/item/misc_weapons.dm [hunk 1]
```diff
@@ -224,7 +224,7 @@
 		boutput(user, "<span class='notice'>The sword is now active.</span>")
 		hit_type = DAMAGE_CUT
 		stamina_damage = active_stamina_dmg
-		if(ishuman(user))
+		if(ishuman(user) && !ON_COOLDOWN(src, "playsound_on", 2 SECOND))
```
Comment (ZeWaka) [inline code/obj/item/misc_weapons.dm:227]:
```suggestion
		if(ishuman(user) && !ON_COOLDOWN(src, "playsound_on", 2 SECONDS))
```

File: code/obj/item/misc_weapons.dm [hunk 2]
```diff
@@ -244,7 +244,7 @@
 		boutput(user, "<span class='notice'>The sword can now be concealed.</span>")
 		hit_type = DAMAGE_BLUNT
 		stamina_damage = inactive_stamina_dmg
-		if(ishuman(user))
+		if(ishuman(user) && !ON_COOLDOWN(src, "playsound_off", 2 SECOND))
```
Comment (ZeWaka) [inline code/obj/item/misc_weapons.dm:247]:
```suggestion
		if(ishuman(user) && !ON_COOLDOWN(src, "playsound_off", 2 SECONDS))
```

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**